src/deleteToList.js

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line import/no-cycle
+import { stock, displayToDoList } from './index.js';
+
+export default (e) => {
+  stock = stock.filter((x) => x.index !== Number(e.target.parentElement.id - 1));
+  for (let i = 0; i < stock.length; i += 1) {
+    stock[i].index = i + 1;
+  }
+  displayToDoList(stock);
+  localStorage.setItem('stock', JSON.stringify(stock));
+};
```

Comment:
- [ ] It seems there is a bug when a user tries to remove a single task, instead, all tasks are removed from screen, but no task is removed from the tasks in localStorage. Let's try to improve upon this behavior. :sweat_smile: 

![todo_list_no_delete](https://user-images.githubusercontent.com/79114233/144712192-82e54e55-dd02-45d9-a15b-b9a8a08d6112.gif)